.flowconfig

```diff
@@ -1,7 +1,5 @@
 [ignore]
 
-.*/node_modules/fbjs/*
```

Comment:
This change only affects the project dev environment. It was removed as flow addressed the issue. Upgrade your version of flow, you are quite behind.